app/src/main/java/org/thoughtcrime/securesms/linkpreview/LinkPreviewUtil.java

```diff
@@ -167,6 +145,21 @@ public static boolean isValidMimeType(@NonNull String url) {
     return new OpenGraph(openGraphTags, htmlTitle, faviconUrl);
   }
 
+  private static void extractTags(@NonNull HtmlDecoder htmlDecoder, Map<String, String> openGraphTags, Matcher articleMatcher) {
```

Comment:
we should probably rename `articleMatcher` to be more generic like `matcher` so it makes sense for the `openGraphMatcher` as well as the `articleMatcher` or whichever else it might be used by in future